modules/installation-guide/examples/checluster-properties.adoc

```diff
@@ -127,7 +127,7 @@ workspacePVCStorageClassName: Storage class for the Persistent Volume Claims ded
 :=== 
  Property: Description 
 ingressClass: Ingress class that will define the which controller will manage ingresses. Defaults to `nginx`. NB\: This drives the `kubernetes.io/ingress.class` annotation on Che-related ingresses.
-ingressDomain: Global ingress domain for a Kubernetes cluster. This MUST be explicitly specified\: there are no defaults.
+ingressDomain: Global ingress domain for a {orch-name} cluster. This MUST be explicitly specified\: there are no defaults.
```

Comment:
Since this section of the `CheCluster` custom resource is for raw Kubernetes clusters, (non-OpenShift), it might make sense to keep `{kubernetes]` here.

Reply:
By the way, it seems there's an error at [line 124](https://github.com/eclipse/che-docs/pull/1888/files#diff-eb21f1372e73e4cac48f5299e610e09d815c3d10cc59a21dbb8efdd092b45d1aR124). It should be `Kubernetes` as well afaik. The current docs are misleading here afaict.

Reply:
Do you mean the name of the parameter is `kubernetes` instead of `k8s`?